desk/ted/commit.hoon

```diff
@@ -0,0 +1,45 @@
+/-  spider
+/-  *herd
+/+  sio=strandio
+=,  strand=strand:spider
+|^  ted
++$  fuse-source  [who=ship des=desk ver=$@(%trak case)]
+++  ted
+  ^-  thread:spider
+  |=  arg=vase
+  =/  m  (strand ,vase)
+  =/  arguments
+    !<  (unit [@tas @tas])  arg
+  =/  [dev=@tas build=@tas]  (need arguments)
+  ;<  our=@p  bind:m  get-our:sio
+  ;<  ~  bind:m
+    %:  poke:sio  [our %hood]
+      [%kiln-fuse !>([build ~])]
+    ==
+  ;<  ~  bind:m
+    (poke:sio [our %hood] [%kiln-commit !>([dev %.n])])
+  ;<  ~  bind:m  (sleep:sio ~s2)
```

Comment:
what's this for? if we need it, possible to make the wait time shorter?

Reply:
Without the sleep, I ran into some issues where the commit wasn't finished when the fuse started. One second is probably enough though.